axltempl/lando.py

```diff
@@ -56,6 +56,13 @@ def generateLandoFiles(name, docroot, cache):
         os.mkdir(".lando")
     util.copyPackageFile("files/lando/php.ini", ".lando/php.ini")
 
+    # Generate lando development override configuration.
+    dir_default = f"{docroot}/sites/default"
+    if not os.path.exists(dir_default):
+        util.writeError(
+            f"The \"{dir_default}\" directory is missing. Unable to generate lando override configuration files.")
```

Comment:
Shall we also say `"This is probably due to composer installation failure (or you specified --no-install). Run init-lando after running composer install"`?